src/shipit_code_coverage_backend/requirements.txt

```diff
@@ -0,0 +1,10 @@
+-e ./../../lib/cli_common[taskcluster,log] #egg=mozilla-cli-common
+-e ./../../lib/backend_common[log,security,cors,api,auth,db,cache] #egg=mozilla-backend-common
+
+aiohttp
+async-lru
+cachetools
+redis
+rq
+gunicorn
+psycopg2
```

Comment:
Not using PostgreSQL (at least for now), so let's add this only if we end up needing it.